### PR TITLE
Fix mso_schema_template_anp_epg if only bd or vrf is provided

### PR DESCRIPTION
**Pull request by lhercot**

### CODE REVIEW

**Review by shrsr — DISMISSED** [review]
LGTM

File: vendor/github.com/ciscoecosystem/mso-go-client/client/client.go [hunk 1]
```diff
@@ -74,6 +75,12 @@ func Platform(platform string) Option {
 	}
 }
 
+func SkipLoggingPayload(skipLoggingPayload bool) Option {
```
Comment (xinyuezhao) [inline vendor/github.com/ciscoecosystem/mso-go-client/client/client.go:78]: what is this func for?

**Review by sajagana — APPROVED** [review]
LGTM!

File: examples/schema_template_anp_epg/main.tf [hunk 2]
```diff
@@ -13,25 +13,52 @@ provider "mso" {
...
+  display_name  = "epg1"
+  bd_name       = mso_schema_template_bd.bd1.name
+  # bd_template_name           = mso_schema_template_bd.bd1.schema_id
+  # bd_schema_id               = mso_schema_template_bd.bd1.template_name
+  vrf_name = mso_schema_template_vrf.vrf1.name
+  # vrf_schema_id              = mso_schema_template_vrf.vrf1.schema_id
+  # vrf_template_name          = mso_schema_template_vrf.vrf1.template
+}
+
+resource "mso_schema_template_bd" "bd1" {
```
Comment (anvitha-jain) [inline examples/schema_template_anp_epg/main.tf:48]: mso_schema_template_bd and mso_schema_template_vrf should go before mso_schema_template_anp_epg also we can remove the commented lines if not necessary.

File: examples/schema_template_anp_epg/main.tf [hunk 3]
```diff
@@ -13,25 +13,52 @@ provider "mso" {
...
+}
+
+resource "mso_schema_template_anp" "anp1" {
+  schema_id    = mso_schema.schema1.id
+  template     = mso_schema.schema1.template_name
+  name         = "anp1"
+  display_name = "anp1"
+}
+
 resource "mso_schema_template_anp_epg" "anp_epg" {
```
Comment (anvitha-jain) [inline examples/schema_template_anp_epg/main.tf:34]: add another mso_schema_template_anp_epg "anp_epg_2" with all the parameters the resource will accept.

**Review by anvitha-jain — CHANGES_REQUESTED** [review]

File: vendor/github.com/ciscoecosystem/mso-go-client/client/client.go (hunk 1 above)
Comment (lhercot) [reply vendor/github.com/ciscoecosystem/mso-go-client/client/client.go:78]: It is used to set the value of the variable. It is just an accessor. I copied it from ACI code.

File: examples/schema_template_anp_epg/main.tf (hunk 2 above)
Comment (lhercot) [reply examples/schema_template_anp_epg/main.tf:48]: The order does not matter in Terraform but for ease of reading I have made the change.

**Review by anvitha-jain — APPROVED** [review]
LGTM

**Pull request merged**